docs/conf.py

```diff
@@ -36,23 +34,19 @@
 author = f"{project} project contributors"
 copyright = author  # pylint:disable=redefined-builtin
 
-# fmt: off
+NAVIGATOR_VERSION = version("ansible_navigator")
+
 # The short X.Y version
-version = ".".join(
```

Comment:
This is a Sphinx setting. Don't lose it. Although, I'm still against breaking what setuptools-scm provides. If you want to reuse the installed dist metadata, you'd have to (1) integrate setuptools-scm into packaging first and then (2) somehow solve the self-reinstallation on every local build in an optimal way (without `tox -r`).